src/5e-SRD-Monsters.json

```diff
@@ -10503,12 +11231,17 @@
         "spellcasting": {
           "level": 4,
           "ability": {
+            "index": "5",
             "name": "WIS",
-            "url": "http://www.dnd5eapi.co/api/ability-scores/5"
+            "url": "/api/ability-scores/5"
```

Comment:
This url also doesn't match. Might want to do a regex for:
```
"url": "/api/[a-z\-]+/[0-9]+"
```
That way we can see if anything else got missed by accident.

Reply:
This can be done in a separate PR though.